api/rest/version.go

```diff
@@ -222,7 +222,7 @@ func listVersions(request *restful.Request, response *restful.Response) {
 }
 
 //
-// POST: /api/v0.1/:uid/versions/:versionID/cancelbuild
+// POST: /api/v0.1/:uid/versions/:versionID/cancelbuildcancelbuild
```

Comment:
This has no error, no need to change. Right?

Reply:
the word `cancelbuild `  repeat twice, it's a small mistake?
![image](https://user-images.githubusercontent.com/29789939/29747013-ebe8c712-8b1e-11e7-980b-c376cf34f4b4.png)